src/base/types/MATModel.jl

```diff
@@ -152,10 +152,13 @@ metabolite_formula(m::MATModel, mid::String) = _maybemap(
 
 Extract metabolite charge from `metCharge` or `metCharges`.
 """
-metabolite_charge(m::MATModel, mid::String) = _maybemap(
-    x -> x[findfirst(==(mid), metabolites(m))],
-    get(m.mat, "metCharge", get(m.mat, "metCharges", nothing)),
-)
+function metabolite_charge(m::MATModel, mid::String)
+    met_charge = _maybemap(
+        x -> x[findfirst(==(mid), metabolites(m))],
+        get(m.mat, "metCharge", get(m.mat, "metCharges", nothing)),
```

Comment:
side idea-- might be useful have this in `keynames` in `src/base/constants.jl`

Reply:
with a `gets` function, that would be best. Also, a lot of these accessors are pretty similar, maybe a macro would be nice here... But not now :)